keyboards/handwired/polykybd/lang/_gen_script_cols.py

```
@@ -0,0 +1,202 @@
+    return ('lit', str(tok))
+
+INDIC=[("bn-IN","bn",0x80,"BENGALI","BENG_DC",0xE120),
+       ("te-IN","te",0x300,"TELUGU","TELU_DC",0xE140),
+       ("ta-IN","ta",0x280,"TAMIL","TAML_DC",0xE160)]
+
+for langname, key, off, PFX, DCPFX, pua_base in INDIC:
+    cm=CMAP[key]
+    dc_marks=set()                 # target mark codepoints used -> composite
+    def conv(tok):
+        p=parse_deva(tok)
+        if p is None: return None
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)